vic/drivers/image/src/get_global_domain.c

```diff
@@ -83,6 +83,18 @@ get_global_domain(char          *nc_name,
         initialize_location(&(global_domain->locations[i]));
     }
 
+    // if MASTER_PROC
+    global_domain->locations_grid = (location_grid_struct *)
+                                    malloc(
+        global_domain->n_ny * global_domain->n_nx *
+        sizeof(location_grid_struct));
```

Comment:
I changed it into (made also a calloc of it):
     `global_domain->locations_grid = calloc(
        global_domain->n_ny * global_domain->n_nx, sizeof(*global_domain->locations_grid));`
I just used the same syntax as a few lines earlier eg line 76. Why do these one still use the old cast syntax? Or just because you didn't push the last commit to github yet?